topics/imaging/tutorials/tutorial-CP/tutorial.md

```diff
@@ -0,0 +1,633 @@
+>       - *"Assign a name to"*: `Give every image the same name` 
+>       - *"Name to assign these images"*: `DNA` 
+>       - *"Select the image type"*: `Grayscale image` 
+>           - *"Set intensity range from"*: `Image metadata` 
+>    - Groups
+>       - *"Do you want to group your images?"*: `Yes, group the images` 
+>       - *"param"*: `field1` 
+>
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The images downloaded from the IDR are named following the pattern: `plate__imageID__cropX__cropY__cropWidth__cropHeight`. These fields indicate to which plate the image belongs, what is the identifier of the image in the IDR, and the 4 cropping parameters selected. In our case, the upper-left corner (X, Y) and the width and height from there. We have a total of 6 metadata values encoded in the name of the file, separated by `__`. The pattern to extract our metadata from the file name properly is, therefore: `field1__field2__field3__field4__field5__field6`. It is important to keep in mind that, later in the analysis, our `plate` will be `field1`, `imageID` will be `field2`, `cropX` will be `field3`, etc. for CellProfiler.
+>    {: .comment}
+>
+{: .hands_on}
+
+# Object segmentation
+
+## Segment nuclei
+
+Since we are interested in segmenting the nucleoli, you may wonder why we need to segment nuclei first. There are several reasons for that: 
+
+- Get the nuclei features. The intensity, size, shape, number of nucleoli per nucleus, etc. can be informative to study the nucleoli.
+
+- Avoid the detection of wrong spots. In a first segmentation, we detect the nuclei, while in the second pass we will segment the holes (nucleoli). The holes need to fall inside the nuclei, and hence the importance of having them segmented too.
+
+In the first step, we will identify the nuclei that are complete, meaning that they are not touching the borders of the image.
+
+> ### {% icon hands_on %} Hands-on: Segment nuclei that are complete within the boundaries of the image
+>
+> 1. **IdentifyPrimaryObjects** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **Starting Modules** {% icon tool %})
+>    - *"Use advanced settings?"*: `Yes, use advanced settings`
+>        - *"Enter the name of the input image (from NamesAndTypes)"*: `DNA`
+>        - *"Enter the name of the primary objects to be identified"*: `Nuclei`
+>        - *"Typical minimum diameter of objects, in pixel units (Min)"*: `15`
+>        - *"Typical maximum diameter of objects, in pixel units (Max)"*: `200`
+>        - *"Discard objects outside the diameter range?"*: `Yes`
+>        - *"Discard objects touching the border of the image?"*: `Yes`
+>        - *"Threshold strategy"*: `Global`
+>            - *"Thresholding method"*: `Otsu`
+>                - *"Two-class or three-class thresholding?"*: `Two classes`
+>                - *"Threshold correction factor"*: `0.9`
```

Comment:
Why 0.9?

Reply:
I've added a sentence explaining that this is a bit arbitrary and needs to be tuned by trial and error :/